test/usecases/table/simulate.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from neuron import h, gui
+
+s = h.Section(name="soma")
+s.L = 10
+s.diam = 10
+s.insert("tbl")
+
+h.k_tbl = -0.1
+h.d_tbl = -40
```

Comment:
Not covered by the test is the feature that the table updates if `k` or `d` change.

Reply:
Thanks, added it now. Note that I don't vary the parameters very much since the interpolation becomes inaccurate, but it seems to actually respond properly to the change in values.